app/views/projects/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_for @project do |f| %>
+  <%= render 'shared/error', object: @project %>
+  <p>
```

Comment:
`<p>` ?. Please wrap it in the proper bootstrap form structure (form-group -> form-control)